Makefile

```diff
@@ -1,7 +1,9 @@
 # everything run by the continuous integration script on GitHub
-ci:
+ci: clean_notebooks
```

Comment:
`make ci` should only run tests, and not modify any code. (eg. we run `black --check` for this purpose).